src/platform/qt/Window.cpp

```diff
@@ -91,15 +92,34 @@
 
 using namespace QGBA;
 
+namespace QGBA {
+
+class WindowPopups {
```

Comment:
By moving these into a private class, I was able to reduce the dependency graph in the header files, so changes to `PopupManager.h` don't cause a dozen files to rebuild.